src/Network/Network.Test/ScenarioTests/AzureFirewallPolicyTests.ps1

```diff
@@ -1558,4 +1558,59 @@ function Test-AzureFirewallPolicyBasicSku {
         # Cleanup
         Clean-ResourceGroup $rgname
     }
+}
+<#
+.SYNOPSIS
+Tests AzureFirewall Policy Explicit Proxy
+#>
+function Test-AzureFirewallPolicyExplicitProxyCRUD {
+    $rgname = Get-ResourceGroupName
+    $azureFirewallPolicyName = Get-ResourceName
+    $resourceTypeParent = "Microsoft.Network/FirewallPolicies"
+    $location = "westus2"
+    $vnetName = Get-ResourceName
+    $pacFile ="https://packetcapturesdev.blob.core.windows.net/explicit-proxy/pacfile.pac?sp=r&st=2022-06-02T21:14:54Z&se=2022-07-15T05:14:54Z&spr=https&sv=2021-06-08&sr=b&sig=VqX7Jfqb0P2HhuoDFDCeGLHvtM65Tu8lpkV96kCWZn0%3D"
+   
+    try {
+
+        # Create the resource group
+        $resourceGroup = New-AzResourceGroup -Name $rgname -Location $location -Tags @{ testtag = "testval" }
+
+        $explicitProxySettings = New-AzFirewallPolicyExplicitProxy -EnableExplicitProxy  -HttpPort 85 -HttpsPort 121 -EnablePacFile  -PacFilePort 122 -PacFile $pacFile
+
+        # Create AzureFirewallPolicy (with Explicit Proxy Settings)
+        $azureFirewallPolicy = New-AzFirewallPolicy -Name $azureFirewallPolicyName -ResourceGroupName $rgname -Location $location -ExplicitProxy $explicitProxySettings
+
+        # Get AzureFirewallPolicy
+        $getAzureFirewallPolicy = Get-AzFirewallPolicy -Name $azureFirewallPolicyName -ResourceGroupName $rgname
+       
+
+        #verification
+        Assert-AreEqual $rgName $getAzureFirewallPolicy.ResourceGroupName
+        Assert-AreEqual $azureFirewallPolicyName $getAzureFirewallPolicy.Name
+        Assert-NotNull  $getAzureFirewallPolicy.Location
+        Assert-AreEqual (Normalize-Location $location) $getAzureFirewallPolicy.Location
+        Assert-NotNull  $getAzureFirewallPolicy.ExplicitProxy
```

Comment:
test failed at this line

Reply:
It is because the ExplicitProxy requires latest SDK release particularly for Microsoft.Azure.Management.Network (current version is 22.0.0 released on Monday, April 25, 2022 (4/25/2022)) since latest required Explicit Proxy features are not included in this release.  [Updated Explicit proxy settings by adding one boolean field to it](https://github.com/Azure/azure-rest-api-specs/pull/19011) and [Updated ExplicitProxySettings to ExplicitProxy on Firewall Policy](https://github.com/Azure/azure-rest-api-specs/pull/19299) are required in the latest Network SDK so this test passes. For local testing purposes, created local Network SDK packages and tested this.